### DISCUSSION TITLE
EJBCA VA for external certificates

### DISCUSSION THREAD

**Discussion by pr2627** [Keyfactor/ejbca-ce#332] [Polls - Free text questions]
Hoping someone can answer a question of function.  Does anyone know if the VA for EJBCA can provide status for certificates from external CAs?  We are aware it provides status for certificates from EJBCA, itself.  Thanks

**Comment by primetomas** [comment]
Yes the OCSP responder can answer for any CA. It just needs an OCSP signing certificate from this CA, and have the VA database populated with revocation information, something that can be done by importing CRLs (which can be done automatically).

https://doc.primekey.com/ejbca/ejbca-operations/ejbca-operations-guide/ca-operations-guide/ocsp-responder-management
https://doc.primekey.com/ejbca/ejbca-operations/ejbca-ca-concept-guide/remote-authenticators-overview/ocsp-responders

**Reply by pr2627** [reply]
thank you for your prompt response.  Much appreciated.
